Cognigy/WebchatWidget
# Add 'Opens in new tab' screen reader only helper for all urls

This PR adds the screen reader phrase 'Opens in new tab' for all links. Additionaly, all links will now get aria role="link" and a tabIndex of 0, so that the links can get keyboard focus and the screen reader will announce its purpose correctly.

Success Criteria:
- All links should receive keyboard focus and should have a visible focus style
- All links should be activated on pressing enter
- All links should have 'Opens in new tab', along with the original link title. 

To test:
- Add Say node and configure the default tab to use 'Text with Buttons'. Add Button of type 'URL'.
- Add Say node and configure the default tab to use 'List'. Add an item with a default action url.
- Add Say node and configure the default tab to use 'Gallery'. Add a card with button of type 'URL'.
- Add Say node and configure the webchat tab to use 'Gallery' as output. Add an element with default action url.
- Add Say node and configure the webchat tab to use 'List' as output and enable large top element. Add an element with default action url.
All these above mentioned URLs should pass the success criteria. Also, please think of any other ways to add a URL/ URL button and check the accessibility.

NOTE: New wrapper `div` with role listitem is added to `MessengerListTemplateElement.tsx `
